wolfssl/wolfcrypt/types.h

```diff
@@ -493,6 +493,63 @@
             #endif /* _MSC_VER || __CYGWIN__ || __MINGW32__ */
         #endif /* USE_WINDOWS_API */
 
+        /* XVASPRINTF is used in ssl.c for wolfSSL_BIO_printf */
+        #ifdef __GNUC__
+            #define VASPRINTFCHECK __attribute__((format(printf, 2, 0)))
+        #else
+            #define VASPRINTFCHECK
+        #endif /* __GNUC__ */
+        #if (defined(sun) || defined(__sun) || defined(_AIX))
+            #include <stdarg.h>
+            VASPRINTFCHECK static WC_INLINE
+            int xvasprintf(char **ret, const char *format, va_list args)
```

Comment:
Is types.h really the best location for this implementation?   What about wc_port?

Reply:
`WC_INLINE` is undefined in `wc_port.h`.

Reply:
Since `XVASPRINTF` is only being used in one place I could simply modify `wolfSSL_BIO_printf`.